source/manual/access-to-licensing-for-third-parties.html.md

```diff
@@ -0,0 +1,58 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Access to Licensify for Third Parties
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+---
+
+[Licensify](/manual/licensing.html) is a GOV.UK application which is usually supported by third parties.
+
+This document explains how to access the Licensify infrastructure as a third party, to perform various maintenance tasks.
+
+## Accessing the logs
+
+Licensify uses an Elasticsearch / Logstash / Kibana system hosted by Logit.io for its logs.
+
+If you haven't already got access to GDS' Logit account, you'll need to ask a GOV.UK developer to set you up.
+
+Once you have an account, you can [follow these instructions to sign in](https://reliability-engineering.cloudapps.digital/logging.html#content).
```

Comment:
Recommend amending link text to "[follow the Reliability Engineering documentation instructions to sign in to Logit]"

Link text must be very explicitly clear for accessibility.